Exec/RegTests/ConvGaussian/GNUmakefile

```diff
@@ -18,10 +18,17 @@ VERBOSE         = FALSE
 TINY_PROFILE = FALSE
 #USE_WBAR = TRUE
 
-DEFINES+=-DALWAYS_NEW_J
+#DEFINES+=-DALWAYS_NEW_J
 
 #######################
-USE_SUNDIALS_3x4x = FALSE
+USE_SUNDIALS_PP = TRUE
+ifeq ($(USE_SUNDIALS_PP), TRUE)
+  # use KLU sparse features -- only useful if CVODE is used on CPU
+  USE_KLU_PP = FALSE
+  # Activates GPU version
+  USE_CUDA_SUNDIALS_PP = FALSE
```

Comment:
isn't it the case that ``USE_CUDA_SUNDIALS_PP`` is really just `USE_CUDA``?

Reply:
That is something that need to be cleaned up: USE_CUDA should automatically trigger USE_CUDA_SUNDIALS_PP (or this flag should simply be removed) and USE_SUNDIALS_PP should always be ON now that DVODE is no longer supported (or again, simply removed ...).

Reply:
I think we should remove ``USE_CUDA_SUNDIALS_PP`` partly because it's a bit obscure, but mostly because it requires that you remember to set both that and ``USE_CUDA`` when switching between CPU and GPU.

Reply:
I can go through PP and remove it, make it USE_CUDA

Reply:
It raises a more fundamental question though, should we keep DVODE ?

Reply:
I think we can toss DVODE.  If there is a reason to bring it back, we will have to wrap it to look like a C function, which is doable, but a waste of time to do without a good reason.  Does that mean we get to remove ``USE_SUNDIALS_PP`` as well?

Reply:
In that case yes